Automattic/vip-go-mu-plugins
# Update VaultPress to version 2.1.1

##  Description

Updating VaultPress 1.9.10 -> 2.1.1

1. Remove VP: `rm -rf vaultpress`
2. Download the zip of the latest version:
~~https://github.com/Automattic/vaultpress/releases/tag/2.1.1~~
Download WordPress.org version: https://wordpress.org/plugins/vaultpress/

curl to download on Sandbox:
~~`curl https://codeload.github.com/Automattic/vaultpress/zip/2.1.1 > vaultpress.zip`~~
`curl --output vaultpress.zip https://wordpress.org/plugins/vaultpress/`

3. On sandbox:
`unzip vaultpress.zip`
`rm vaultpress.zip`
(if you need to change the name: `mv vaultpress-2.1.1 vaultpress`)

Locally: Unzip, rename, and move back to `mu-plugins` directory

4. Update the plugin headers in `vaultpress.php`  with the new version number
5. `git diff` to see the changes

## Checklist

Please make sure the items below have been covered before requesting a review:


Seeing the deprecated errors requires PHP 7.4, but most sites are still on PHP 7.3.16

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Go to `wp-admin` > `Jetpack` > `VaultPress`
1. Verify everything is working as expected + VaultPress version is 2.1
1. Go to site & verify everything is working
